cpu/efm32/periph/timer.c

```diff
@@ -52,18 +65,42 @@ static inline bool _is_wtimer(timer_t dev)
 #endif
 }
 
-int timer_init(tim_t dev, unsigned long freq, timer_cb_t callback, void *arg)
+/**
+ * @brief   Check whether dev is using a LETIMER device
+ */
+static inline bool _is_letimer(timer_t dev)
 {
-    TIMER_TypeDef *pre, *tim;
+#if defined(LETIMER_COUNT) && LETIMER_COUNT > 0
+    return ((uint32_t) timer_config[dev].timer.dev) == LETIMER0_BASE;
+#else
+    (void) dev;
```

Comment:
Very nit-picky: `_is_wtimer` has an additional newline between `(void) dev` and `return false`. Either add one here, or remove that one from `_is_wtimer` ;-)